fastn-core/src/commands/serve.rs

```diff
@@ -587,9 +587,20 @@ async fn handle_endpoints(
     tracing::info!("url: {}", url);
 
     if url.starts_with("wasm+proxy://") {
+        let app_mounts = match config.package.app_mounts() {
```

Comment:
Make this `config.app_mounts()`.

Reply:
done in https://github.com/fastn-stack/fastn/pull/2059/commits/b74a3d1983a1ec4e5b50157d1f73e8810a1b400b

Reply:
In case you are curious, we did this because "app_mounts" only makes sense for "top level package", which is `config.package`, but the `Package` type is used in a bunch of other places as well, and if someone called `Package.app_mounts()` for non "top level package" it would be garbage, so we want that to be compile time impossibility, by having `Config.app_mounts()` which will always make sense.